docs/README.rst

```diff
@@ -66,6 +66,8 @@ Within the cluster, the allocated NodePort load balances traffic to all pods.
 
    F5 does not recommend making configuration changes to objects in any partition managed by the |kctlr| via any other means (for example, the configuration utility, TMOS, or by syncing configuration with another device or service group). Doing so may result in disruption of service or unexpected behavior.
 
+   In order to support feature compatibility with the Openshift F5 Router, the controller allows one exception to this recommendation.  When using named virtual servers for :ref:`Openshift routes <openshift route configs>`, the user can modify the controller's behavior so that it merges its desired settings with a pre-existing virtual server(s). See `Manage OpenShift Routes`_ for more information.
```

Comment:
My recommendation would be to omit "In order to support feature compatibility with the Openshift F5 Router," from the sentence and just start at "The controller allows ...".